airflow/bin/cli.py

```diff
@@ -859,10 +859,12 @@ def serve_logs(filename):  # noqa
             mimetype="application/json",
             as_attachment=False)
 
+    WORKER_LOG_SERVER_BIND_IP = \
+        conf.get('celery', 'WORKER_LOG_SERVER_BIND_IP')
     WORKER_LOG_SERVER_PORT = \
         int(conf.get('celery', 'WORKER_LOG_SERVER_PORT'))
     flask_app.run(
-        host='0.0.0.0', port=WORKER_LOG_SERVER_PORT)
+        host=WORKER_LOG_SERVER_BIND_IP, port=WORKER_LOG_SERVER_PORT)
```

Comment:
It's unlikely, but is it possible (via another option?) to specify an interface rather than an IP?

If that's easy to do (i.e. it's in flask) this would be a useful addition. If not this is still an improvement